scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/utils/TaskKiller.java

```diff
@@ -63,19 +60,10 @@ public synchronized boolean wasKilled() {
     private void interruptRepeatedly() {
         while (threadToKill.isAlive()) {
             threadToKill.interrupt();
-            threadInterruptSleeper.sleep();
-        }
-    }
-
-    /**
-     * Interrupts and gives the thread time to cleanup
-     */
-    private void interruptAndWaitCleanupTime() {
-        threadToKill.interrupt();
-        int iterationsToHitWaitingTimeLimit =
-                SECONDS_TO_WAIT_UNTIL_AGGRESSIVELY_INTERRUPT_TASK * THREAD_INTERRUPT_SLEEP_TO_SECONDS_MULTIPLIER;
-        for (int i = 0; threadToKill.isAlive() && i < iterationsToHitWaitingTimeLimit; i++) {
-            threadInterruptSleeper.sleep();
+            try {
+                Thread.sleep(10);
+            } catch (InterruptedException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS00108)